fmt/printf.h

```diff
@@ -588,6 +600,8 @@ FMT_VARIADIC(int, printf, CStringRef)
   \endrst
  */
 inline int fprintf(std::ostream &os, CStringRef format_str, ArgList args) {
+  std::mutex mtx;
+  std::unique_lock<std::mutex> lck(mtx);
```

Comment:
27.1.3 Thread safety [iostreams.thread-safety]

Concurrent access to a stream object [string.streams, file.streams], stream buffer object [stream.buffers], or C Library stream [c.files] by multiple threads may result in a data race [intro.multithread] unless otherwise specified [iostream.objects]. [Note: Data races result in undefined behavior [intro.multithread].